#!/usr/bin/env python
PACKAGE = "test_pkg"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("threshold_param", bool_t, 0, "threshold parameter", False)
gen.add("flag_param", int_t, 0, "flag parameter", 0, 0, 2)
gen.add("save_param",   bool_t,    0, "saving parameter", False)
gen.add("t1min_param",    int_t,    0, "t1min parameter", 0,  0, 260)
gen.add("t1max_param",    int_t,    0, "t1max parameter", 0,  0, 260)
gen.add("t2min_param",    int_t,    0, "t2min parameter", 0,  0, 260)
gen.add("t2max_param",    int_t,    0, "t2max parameter", 0,  0, 260)
gen.add("t3min_param",    int_t,    0, "t3min parameter", 100,  0, 260)
gen.add("t3max_param",    int_t,    0, "t3max parameter", 255,  0, 260)

exit(gen.generate(PACKAGE, "test_pkg", "gate"))
